src/hooks/useSetSentry.ts

```diff
@@ -0,0 +1,34 @@
+import { setContext, setUser } from "@sentry/nextjs"
+import { useWalletSelector } from "@src/providers/WalletSelectorProvider"
+import { useEffect } from "react"
+import { useAccount } from "wagmi"
+
+export const useSentrySetUser = ({ userAddress }: { userAddress?: string }) => {
+  useEffect(() => {
+    if (userAddress) {
+      setUser({ id: userAddress })
+    }
```

Comment:
Should we clear user id when they sign out?

Reply:
Yeap